package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@autotelic/gql-schema-normalizer",
+  "version": "0.0.0",
+  "description": "Normalize GraphQL string schemas.",
+  "main": "index.js",
+  "files": [],
+  "scripts": {
+    "lint": "standard",
+    "test": "tap --100 -j1 -R classic",
+    "validate": "npm run lint && npm run test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/autotelic/gql-schema-normalizer.git"
+  },
+  "keywords": [
+    "GraphQL",
+    "Schema",
+    "Tools",
+    "Utils"
+  ],
+  "author": "Holden Whitehead <holden@autotelic.com>",
+  "license": "MIT",
```

Comment:
Can we add a LICENSE.md file?

Reply:
The repo was initialized with a LICENSE, README.md, and .gitignore -> https://github.com/autotelic/gql-schema-normalizer/blob/main/LICENSE